.github/workflows/actions.yml

```diff
@@ -12,112 +12,15 @@ permissions:
-
-      - name: Upload the report json
-        uses: actions/upload-artifact@v3
-        with:
-          name: report
-          path: report.json
-
-      - name: Download the base report
-        uses: dawidd6/action-download-artifact@v2
-        if: github.event_name == 'pull_request'
-        continue-on-error: true
-        with:
-          workflow: actions.yml
-          name: report
-          commit: ${{ github.event.pull_request.base.sha }}
-          path: base-report
-
-      - name: Generate the report template
-        if: github.event_name == 'pull_request'
-        run: |
-          echo "Contract comparison - from {{ .base }} to {{ .head }}" > report.md
-          if [ ! -f base-report/report.json ]
-          then
-              echo ":warning: Warning: Could not download the report for the base branch. Displaying only the report for the current branch. :warning:" >> report.md
-              mxpy contract report --compare report.json --output-format github-markdown --output-file report-table.md
-          else
-              mxpy contract report --compare base-report/report.json report.json --output-format github-markdown --output-file report-table.md
-          fi
-          cat report-table.md >> report.md
-
-      - name: Render the report from the template
-        id: template
-        uses: chuhlomin/render-template@v1
-        if: github.event_name == 'pull_request'
-        with:
-          template: report.md
-          vars: |
-            base: ${{ github.event.pull_request.base.sha }}
-            head: ${{ github.event.pull_request.head.sha }}
-
-      - name: Upload the report markdown
-        uses: actions/upload-artifact@v3
-        if: github.event_name == 'pull_request'
-        with:
-          name: report-markdown
-          path: report.md
-
-      - name: Find the comment containing the report
-        id: fc
-        uses: peter-evans/find-comment@v2
-        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          comment-author: "github-actions[bot]"
-          body-includes: "Contract comparison"
-
-      - name: Create or update the report comment
-        uses: peter-evans/create-or-update-comment@v2
-        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
-        with:
-          comment-id: ${{ steps.fc.outputs.comment-id }}
-          issue-number: ${{ github.event.pull_request.number }}
-          body: ${{ steps.template.outputs.result }}
-          edit-mode: replace
+  contracts:
+    name: Contracts
+    uses: multiversx/mx-sc-actions/.github/workflows/contracts.yml@v3.0.0
+    with:
+      rust-toolchain: nightly-2023-12-11
+      path-to-sc-meta: framework/meta
+      mx-scenario-go-version: v2.1.0-alpha
```

Comment:
Good rust version :+1: 

Using meta from the repository itself :+1: 

Tests runner version as before :+1: